docs/source/index.rst

```diff
@@ -57,7 +59,8 @@ the rendered ouput JSON.
 
 The ``base_template_name`` (default ``base.html``) keyword argument allows you
 to provide a base template fitting your site's layout. It should provide
-``title``, ``doc``, and ``toc`` blocks.
+``title``, ``doc``, and ``toc`` blocks. Include the ``{{ doc.toctree|safe }}``
+variable in your template to render a global Table of Contents.
```

Comment:
This is OK, for now, but we can probably expand this a bit...

Reply:
Sounds good